cli/drivers/BasicValetDriver.php

```diff
@@ -44,10 +44,12 @@ public function isStaticFile($sitePath, $siteName, $uri)
      */
     public function frontControllerPath($sitePath, $siteName, $uri)
     {
+        $this->loadServerEnvironmentVariables($sitePath, $siteName);
```

Comment:
Since this seems to be happening in every `frontControllerPath` method, is it possible to instead just run this in `server.php` right before it calls `frontControllerPath`? (I don't know the code super well so there may be a good reason why not...)

Reply:
Technically it probably could.

https://github.com/laravel/valet/blob/0323607850ec051ad4746409ed0992c5f24c37d9/server.php#L134-L147